aula/apps/alumnes/abstract_models.py

```diff
@@ -322,7 +358,97 @@ def aniversari( self, data=None):
     def get_foto_or_default(self):
         foto = self.foto.url if self.foto else static('nofoto.png')
         return foto
-
+    
+    def get_responsables(self, rp1_dni=None, rp2_dni=None, compatible=False):
+        '''
+        Selecciona els responsables de l'alumne que corresponen
+        als dnis indicats.
+        Si no s'aporten els dnis, selecciona els responsables
+        existents de l'alumne, en aquest cas el primer és el preferent.
+        retorna els 2 responsables, poden ser None si no existeixen.
+        '''
+        from django.apps import apps
+        
+        if not bool(rp1_dni) and not bool(rp2_dni):
+            responsables=list(self.responsables.all())
+            #DEPRECATED vvv
+            Responsable=apps.get_model('relacioFamilies', 'Responsable')
+            if not responsables and compatible and not Responsable.objects.exists():
```

Comment:
Aquest `if` té una expressió que només es compleix quan no hi ha cap responsable a la bd, a partir que ja n'hi hagi un es deixa de complir

Reply:
Només crea uns responsables temporals si encara no s'ha fet la nova càrrega de dades d'alumnes amb els responsables diferenciats (not Responsable.objects.exists())
Això només es per compatibilitat mentre no es faci la càrrega nova. Si ja s'ha fet la càrrega, i l'alumne encara no té responsables associats, s'assumeix que es tracta d'un alumne sense responsables.